src/code/z_actor.c

```diff
@@ -389,7 +401,7 @@ void func_8002C124(TargetContext* targetCtx, PlayState* play) {
                     for (i = 0; i < 4; i++) {
                         Matrix_RotateZ(M_PI / 2, MTXMODE_APPLY);
                         Matrix_Push();
-                        Matrix_Translate(entry->unk_0C, entry->unk_0C, 0.0f, MTXMODE_APPLY);
+                        Matrix_Translate(entry->radius, entry->radius, 0.0f, MTXMODE_APPLY);
```

Comment:
hm, technically the real "radius" might be `radius` times sqrt(2) since this translates in both x and y. But that doesn't seem very important